### ISSUE TITLE
Dreadful dirt not spawning.

### ISSUE THREAD

**Issue by wampire3261** [vadis365/Mob-Grinding-Utils#249]
I'm playing better minecraft modpack 1.19.2 I added mob grinding utils myself it was working at first but when decided to build in peaceful and come back to normal the dreadful dirt was not working. I checked everything I could think of like the light level, claimed chunks and even nearby caves in case of capped mob count. I have delightful dirt set up just the same plus lanterns and it works normally. Any assistance would be great.

**Comment by HamTurkey** [comment]
I just updated my ATM8 server to 1.0.16 which includes mob_grinding_utils-1.19.2-0.4.48 and am having the same issue.
Dreadful dirt is no longer spawning mobs after the update.
Thank you.

**Comment by DemonSeedxp** [comment]
Yeah I'm having issues with it not spawning anymore too. Mine seems so be self Illuminating, so much so that it's also preventing standered spawners from working. The only two that it didn't mess with were the blaze and magma cube spawners due to the above avg light tolerance they have.  I don't know what exactly I did to cause it to stop. But I did visit a mineing dimension that I found a pig spawner in, so I guess it likly put the game in peacful while I was there. So it may be the same as the OP mentioned happened to their world.
